nixos/modules/services/hardware/bluetooth.nix

```diff
@@ -86,6 +109,10 @@ in
     environment.systemPackages = [ package ]
       ++ optional cfg.hsphfpd.enable pkgs.hsphfpd;
 
+    environment.etc."bluetooth/input.conf".source =
+      cfgFmt.generate "input.conf" (recursiveUpdate defaults cfg.input);
+    environment.etc."bluetooth/network.conf".source =
+      cfgFmt.generate "network.conf" (recursiveUpdate defaults cfg.network);
```

Comment:
According to #204418 this won't work

> BlueZ loads input.conf from a hardcoded path

We need a symlink in the package, like the one for main.conf